app/build.gradle

```diff
@@ -8,6 +8,7 @@ apply plugin: 'checkstyle'
 android {
     compileSdkVersion 25
     buildToolsVersion "25.0.3"
+    testBuildType "beta"
```

Comment:
@pocmo so I had to specify which build variant the test can run, as it seems that tests can run on only one variant at a time.  without above, it will not apply dependencies from 'androidTestCompile' or 'testCompile' apparently. (https://stackoverflow.com/questions/32484985/android-studio-instrumentation-testing-build-variant)